vaadin-board-row.html

```diff
@@ -55,20 +55,26 @@
        * Calculates flex basis based on colSpan and width
        * @param{Number} colspan value of the row
        * @param{Number} width of the row in px
-       * @param{Number} amount of nodes in the row
+       * @param{Number} amount of columns in the row
        */
-      _calculateFlexBasis(colSpan, width, nodesCount) {
+      _calculateFlexBasis(colSpan, width, colsInRow) {
         let flexBasis = 25 * colSpan;
```

Comment:
I do not see an execution path where this calculation is actually used. Every possible path below overwrites it